src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -29,13 +33,35 @@
     /**
      * Looks at the definition of {@link AbstractProject} and list up the related github repositories,
      * then puts them into the collection.
+     *
+     * @deprecated Use {@link GitHubRepositoryNameContributor#parseAssociatedNames(Job job, Collection result)}
```

Comment:
I think you mean to use the syntax `{@link #parseAssociatedNames(Job, Collection)}`.